react-graphql-apollo-tutorial.md

```diff
@@ -600,7 +602,7 @@ const Profile = () => (
 export default Profile;
 {{< /highlight >}}
 
-Last but not least, you need to make sure that when no data is there yet, that you show something else, because the viewer object is not there.
+Make sure to give some type of visual feedback until you've installed a view layer:
```

Comment:
Better: Last but not least, you need to make sure that when no data is there yet, that you show something else, because the viewer object is not there.

Personally I feel the word "installation" has a different connotation in this context.